fusionauth-netcore-client/src/io/fusionauth/FusionAuthClient.cs

```diff
@@ -3300,4 +3300,24 @@ internal class DefaultRESTClientBuilder : IRESTClientBuilder {
       return new DefaultRESTClient(host);
     }
   }
+
+  public class HttpClientBuilder : IRESTClientBuilder
+  {
+    public HttpClient HTTP_CLIENT;
+
+    public HttpClientBuilder(HttpClient httpClient)
+    {
+      HTTP_CLIENT = httpClient;
+    }
+
+    public IRESTClient build(string host)
+    {
+      if (HTTP_CLIENT.BaseAddress == null)
```

Comment:
Echoing comment from https://github.com/FusionAuth/fusionauth-client-builder/pull/75#discussion_r1680384325

If a host string is passed in, but the BaseAddress of the HTTP_CLIENT is already set, the parameter is effectively ignored. This might be an anti-pattern.